test/unit/calculators/state_pension_age_calculator_test.rb

```diff
@@ -60,6 +60,17 @@ class StatePensionAgeCalculatorTest < ActiveSupport::TestCase
           assert_equal @calculator.state_pension_age, '60 years'
         end
       end
+
+      context 'given a state_pension_date on the 29th of Feb when the pension age is 68' do
```

Comment:
You could move this into the `context` above, because it's closely related.

Reply:
Agreed